src/main/java/com/client/core/email/MailSettings.java

```diff
@@ -1,120 +1,28 @@
 package com.client.core.email;
 
-import java.util.List;
-
 import com.google.common.collect.Lists;
+import lombok.Data;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.List;
 
 /**
  * Object used to represent settings used for emailing.  Values should be provided via a properties file and injected
  * using a Spring {@link org.springframework.beans.factory.config.PropertyPlaceholderConfigurer}, either in
  * applicationContext.xml or using {@link org.springframework.beans.factory.annotation.Value} annotations
  */
+@Data
+@Configuration
 public class MailSettings {
```

Comment:
This could read from properties using `@ConfigurationProperties(prefix = "mail")`

Reply:
Good catch!